### PR TITLE
Globals

### PR DESCRIPTION
**Pull request by apozharski**
Finally got around to adding this. Probably want an example for this as well at some point.

### CODE REVIEW

File: src/FiniteElement.m [hunk 1]
```diff
@@ -563,10 +563,10 @@ function forwardSimulation(obj, ocp, Uk, s_sot, p_stage)
 
             for j = 1:dims.n_s
                 % Multiply by s_sot_k which is 1 if not using speed of time variable
-                [fj, qj] = model.f_x_fun(X_ki{j}, obj.rkStageZ(j), Uk, p_stage);
+                [fj, qj] = model.f_x_fun(X_ki{j}, obj.rkStageZ(j), Uk, p_stage, model.v_global);
```
Comment (nurkanovic) [inline src/FiniteElement.m:566]: why is `v_global` taken out of `model`, but not out of the local workspace (comming e.g. from unfold struct) like the other variables?

File: src/model_reformulation_nosnoc.m [hunk 2]
```diff
@@ -1037,7 +1037,7 @@
     if simple_v0_guess
         v0 = zeros(n_x,1);
     else
-        [v0,~] = (f_x_fun(x0,z0_all,u0,[p_global_val; p_time_var_val(:,1)]));
+        [v0,~] = (f_x_fun(x0,z0_all,u0,[p_global_val; p_time_var_val(:,1)], v0_global));
```
Comment (nurkanovic) [inline src/model_reformulation_nosnoc.m:1040]: hmm, the names of v0 and v0_global are a bit confusing, maybe adding a comment would be instructive.
ones are stage "velocity values", others are global time independent variables

**Review by nurkanovic — COMMENTED** [review]
LGTM except the small comments that you can clarify to me.
Also, it is important to have an example (like the cart pole in nosnoc py) and possibly a test, to really see does it work as it should. :D

File: src/FiniteElement.m (hunk 1 above)
Comment (apozharski) [reply src/FiniteElement.m:566]: I'd like to _eventually_ tear out all (or almost all) of the `unfold_struct` usage. Inside the problem definition classes there is already none.

File: src/model_reformulation_nosnoc.m (hunk 2 above)
Comment (apozharski) [reply src/model_reformulation_nosnoc.m:1040]: Yea a comment for now but I still don't love the `v_global` name as a whole.

Comment (nurkanovic) [reply src/model_reformulation_nosnoc.m:1040]: I am more than open to better suggestions 😁 
We were discussing for a while how to name it and we wanted to be consistent with the global user parameters.
But somewhat we run out of letters.

File: src/FiniteElement.m (hunk 1 above)
Comment (nurkanovic) [reply src/FiniteElement.m:566]: This is s good point actually.
Recently, I discussed with @FreyJo, and we agreed that actually having the prefix to where a function/variable belongs, improves the code readability.

**Review by FreyJo — APPROVED** [review]
LGTM!

**Pull request merged**